test/integration/db.go

```diff
@@ -251,7 +262,6 @@ func DBSetup() func() {
 		up.Do(func() {}) // Trip the sync.Once
```

Comment:
I'm not sure I entirely understand this line, the `sync.Once` seems to be called either way as part of `NeedDB()`

Reply:
If the process doesn't need to do any work because there's an external database, tripping the Once means those functions won't get called.

Reply:
Oh ok, I was conflating the two cases: "GH actions external DB" and just "external DB"